schemas/context/segmentdefinition-expression.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition-expression",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Expression",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Segment expression",
+  "definitions": {
+    "segmentdefinition-expression": {
```

Comment:
Can you please check if we follow any convention to have these in camel case? Reason I am asking is that most of the places I have seen these in camel case. But I am not sure if there is a convention followed or is it by chance.

Reply:
I got inspired from experienceevent, phonenumber, identityitem etc.
<class all lower>-<mixin>

Reply:
Camelcase is the preferred approach. This is outlined in https://github.com/adobe/xdm/blob/master/CONTRIBUTING.md#coding-styleguides

Reply:
@cfraser  this is not property name and it also be same as URL specifying $id. Should it still be came case?

Reply:
I don't think this definition properties -must- be camel case. But field properties should follow the above guidelines.